install/installing-knative-eventing.adoc

```diff
@@ -23,4 +23,8 @@ include::modules/serverless-install-kafka-odc.adoc[leveloffset=+1]
 [id="next-steps_installing-knative-eventing"]
 == Next steps
 
-* If you want to use Knative services you can xref:../install/installing-knative-serving.adoc#installing-knative-serving[install Knative Serving].
+* Learn more about how to use Knative event-driven architecture in xref:../eventing/knative-eventing.adoc#knative-eventing-overview[Knative Eventing].
+
+* If you want to use Knative Eventing with Apache Kafka, see xref:../install/serverless-kafka-admin.adoc#serverless-kafka-admin[Configuring Knative broker for Apache Kafka].
+
+* If you want to use Knative services, see xref:../install/installing-knative-serving.adoc#installing-knative-serving[Installing Knative Serving] and then start xref:../knative-serving/getting-started/serverless-applications.adoc#serverless-applications[Creating serverless applications].
```

Comment:
I won't hold up merging for this but pointing out that you've capitalized here and not in the other file when you added the link. (I think either is fine - I lean more towards no caps because you're linking an action here - but I would be consistent.)